src/test/java/com/yelp/nrtsearch/server/luceneserver/warming/WarmerTest.java

```diff
@@ -127,11 +127,11 @@ public void testWarmFromS3()
 
   @Test
   public void testWarmFromS3_parallel()
-          throws IOException, SearchHandler.SearchHandlerException, InterruptedException {
+      throws IOException, SearchHandler.SearchHandlerException, InterruptedException {
     Path warmingQueriesDir = folder.newFolder("warming_queries").toPath();
     int warmingCountPerQuery = 10;
     try (BufferedWriter writer =
-                 Files.newBufferedWriter(warmingQueriesDir.resolve("warming_queries.txt"))) {
+        Files.newBufferedWriter(warmingQueriesDir.resolve("warming_queries.txt"))) {
```

Comment:
This will create a conflict with the other PR, may be just discard the changes in this file?